stix_shifter_modules/elastic_ecs/entry_point.py

```diff
@@ -11,5 +14,15 @@ def __init__(self, connection={}, configuration={}, options={}):
         if connection:
             self.setup_transmission_basic(connection, configuration)
 
+        basepath = os.path.dirname(__file__)
+        filepath = os.path.abspath(os.path.join(basepath, "stix_translation"))
+
         dialect = 'default'
-        self.setup_translation_simple(dialect_default=dialect, results_translator=ResultTranslator(options, dialect))
\ No newline at end of file
+        query_translator = QueryTranslator(options, dialect, filepath)
+        results_translator = ResultTranslator(options, dialect)
+        self.add_dialect(dialect, query_translator=query_translator, results_translator=results_translator, default_include=True, default=True)
+
+        dialect = 'beats'
+        query_translator = QueryTranslator(options, dialect, filepath)
+        results_translator = ResultTranslator(options, dialect)
+        self.add_dialect(dialect, query_translator=query_translator, results_translator=results_translator, default_include=False, default=False)
```

Comment:
I think default `from_stix_map.json` should be renamed to `ecs_from_stix_map.json`. that way we will have two dialects ecs and beats.

Another thing, should this always construct two queries? if not unless user sets the beats dialects through datasource settings, it will always construct default query which is ecs in this case.